### PR TITLE
[4.0] RavenDB-11598 fixing test

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: test/FastTests/Client/LoadAsync.cs [hunk 1]
```diff
@@ -60,12 +62,26 @@ public class LoadAsync : RavenTestBase
...
+                            {
+                                await Task.WhenAll(load1, load2, load3, load4, load5);
+                            }
+                            catch (InvalidOperationException)
+                            {
+                                throw;
+                            }
+                            catch (Exception)
+                            {
+                                // we do not have a choice, we are accessing session in a concurrent way
```
Comment (ayende) [inline test/FastTests/Client/LoadAsync.cs:81]: So this might still fail, right?

Comment (ppekrol) [reply test/FastTests/Client/LoadAsync.cs:81]: If all 5 of them will throw NRE, yes...

Comment (ayende) [reply test/FastTests/Client/LoadAsync.cs:81]: I wonder if we can make it _not_ race by hitting a slow proxy?

Comment (ayende) [reply test/FastTests/Client/LoadAsync.cs:81]: Probably not worth it

Comment (ppekrol) [reply test/FastTests/Client/LoadAsync.cs:81]: I think we can go with this... and when it fails... remove this test entirely? There are too many places inside Session that can blow when accessed concurrently.

**Review by ayende — APPROVED** [review]

**Pull request merged**